CONTRIBUTING.md

```diff
@@ -44,3 +44,16 @@ install the following:
 
 Then from the command line navigate to the `/docs` directory and call `make
 html`. The HTML pages will be in `_build/html`.
+
+##### Making a release (needs admin rights)
+1. go to your python environment for `pybv`
+1. run `pip install -U setuptools wheel twine`
+1. update the `__version__` variable in `__init__.py`
```

Comment:
do we want to adopt something like `<version>.dev0` for pybv? I've found it makes it easier to keep track of when we're in a dev version vs. when we need to bump versions etc.

Reply:
+1, we are currently using `-dev`, see [here](https://github.com/bids-standard/pybv/blob/f1c84c062b8f1e3596d63b331c5d205d5b9c109e/pybv/__init__.py#L10)